actions/class.Items.php

```diff
@@ -219,6 +219,15 @@ public function editItem()
             $this->setData('isAuthoringEnabled', $hasModel);
 
             $this->setData('formTitle', __('Edit Item'));
+            
+            if($myForm->hasAsyncFileUpload()){
+                $hiddenElement = \tao_helpers_form_FormFactory::getElement('', 'hidden');
```

Comment:
Is it ok that we provide an empty string as the first parameter?

Reply:
Yes. Either we can pass as empty string (Which is used to set the name of the element) or pass an argument to set the name
Here I am going to refactor by passing name as the first param and removing the line 
`$hiddenElement->setName('check-file');`